### PR TITLE
[imager] Cosmetic: Add HandleFilesWithCallback to utils and refactor

### PR DESCRIPTION
**Pull request by leo8a**
Follow up PR of #79, specifically addressing [this comment](https://github.com/openshift-kni/lifecycle-agent/pull/79#discussion_r1405396791).

### CODE REVIEW

File: utils/utils.go [hunk 1]
```diff
@@ -137,3 +138,17 @@ func ReadImageFromStaticPodDefinition(podFile, containerName string) (string, er
 
 	return "", fmt.Errorf("no '%s' container found or no image specified in %s", containerName, podFile)
 }
+
+func FindFilesInFolderAndRun(folder string, action func(string) error) error {
```
Comment (tsorya) [inline utils/utils.go:142]: It was my name proposition  and it is bad as all of them :) Maybe idea for better naming?

Comment (leo8a) [reply utils/utils.go:142]: I'm no better at proposing names, believe me :)

just a few that come from the top of my head, in no preference order (open to suggestions)
```
    WalkOverFiles
    RunOnEachFile
    ExecuteForFiles
    PerformOnFiles
```

Comment (tsorya) [reply utils/utils.go:142]: HandleFilesWithCallback

**Review by donpenney — COMMENTED** [review]
/lgtm

**Pull request merged**